contracts/PriceOracle/PriceOracle.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.7;
+
+import { Ownable } from "../Ownable.sol";
+import { AggregatorV3Interface } from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+import { FullMath } from "../Uniswap/UniswapLib.sol";
+
+contract PriceOracle is Ownable {
+
+    /// @dev Configuration used to return the USD price for the associated cToken asset and base unit needed for formatting
+    /// There should be 1 TokenConfig object for each supported asset, passed in the constructor.
+    struct TokenConfig {
+        // The address of the Compound Token
+        address cToken;
+        // The number of smallest units of measurement in a single whole unit. (e.g. 1e18 for ETH)
+        uint256 baseUnit;
+        // Address of the feed used to retrieve the asset's price
+        address priceFeed;
+    }
+
+    /// @dev Mapping of cToken address to TokenConfig used to maintain the supported assets
+    mapping (address => TokenConfig) tokenConfigs;
```

Comment:
We can actually give names to mappings now.

```suggestion
    mapping (address cToken => TokenConfig cTokenConfig) tokenConfigs;
```

Reply:
I think this isn't available for solidity `0.8.7`. I wanted to use a newer version but think I'd have to upgrade all of the contracts because of hardhat configs. I was trying to avoid changing the other contracts so we don't have a version on UAV in git that's not deployed and being used.